package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "erlang_ls",
+    "version": "0.0.1",
+    "description": "Erlang Language Server",
+    "homepage": "https://github.com/erlang-ls/vscode#readme",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/erlang-ls/vscode.git"
+    },
+    "bugs": {
+        "url": "https://github.com/erlang-ls/vscode/issues"
+    },
+    "main": "./client/out/client.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "vscode:prepublish": "cd client && npm run update-vscode && cd .. && npm run compile",
+        "compile": "cd erlang_ls && rebar3 escriptize && cd ../client && npm run compile",
```

Comment:
We can now replace `rebar3 escriptize` with `make`.

Reply:
Done.